Fakebook.Posts/Fakebook.Posts.DataAccess/Models/Comment.cs

```diff
@@ -10,7 +10,7 @@ public class Comment
         public int PostId { get; set; }
         public Post Post { get; set; }
         public string Content { get; set; }
-        public DateTimeOffset CreatedAt { get; set; }
+        public DateTime CreatedAt { get; set; }
```

Comment:
it's actually the data access layer that should change here, not here. we [almost always want DateTimeOffset](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime) rather than DateTime.

Reply:
If i didnt change it, our test fails because SQLLITE doesn't handle datetimeoffset. Any suggestions on how i can fix that issue?